app/(pages)/ecommerce/_components/EcommercePricingPlansSection/EcommercePricingPeriodToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from "react";
+
+import { ToggleButton } from "components";
+
+type TProps = {
+  activePeriod: string;
+  setActivePeriod: (period: string) => void;
+};
+
+export const EcommercePricingPeriodToggle: FunctionComponent<TProps> = ({
+  activePeriod,
+  setActivePeriod,
+}) => {
+  return (
+    <div className="flex p-1 bg-black rounded-xl w-full max-w-[354px]">
+      <ToggleButton
+        text="Monthly"
+        activePeriod={activePeriod}
+        setActivePeriod={setActivePeriod}
```

Comment:
No need for a `setActivePeriod` it should be a form, please refactor with the usage of the react-hook-form